src/components/CityCard.js

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+
+const CityCard = ({ citydetails, onClick }) => {
+  const { name, state, country } = citydetails;
+
+  const showAlert = () => {
+    const selectedCity = citydetails;
+    onClick(onClick(selectedCity));
```

Comment:
why called onClick two times here? I didn't get this logic

Reply:
Removed